enterprise/admin/forms.py

```diff
@@ -86,7 +87,22 @@ def clean_email_or_username(self):
             return email_or_username
 
         email = email_or_username__to__email(email_or_username)
-        validate_email_to_link(email, email_or_username, ValidationMessages.INVALID_EMAIL_OR_USERNAME)
+        bulk_entry = len(split_usernames_and_emails(email)) > 1
+        if bulk_entry:
+            for email in split_usernames_and_emails(email):
```

Comment:
We can avoid calling `split_usernames_and_emails(email)` multiple times by moving it up to L89 and referencing the output for the `bulk_entry` determination and subsequent loop.